### PR TITLE
Rl lib interface

### PR DESCRIPTION
**Pull request by alexpalms**

### CODE REVIEW

File: stable_baselines3/dict_obs_space.py [hunk 1]
```diff
@@ -1,52 +1,10 @@
...
-    # When using one environment, no need to start subprocesses
-    if num_envs == 1 or not use_subprocess:
-        env = DummyVecEnv([make_sb_env(i) for i in range(num_envs)])
-    else:
-        env = SubprocVecEnv([make_sb_env(i) for i in range(num_envs)])
-
-    return env, num_envs
 
-if __name__ == '__main__':
+if __name__ == "__main__":
```
Comment (discordianfish) [inline stable_baselines3/dict_obs_space.py:7]: Why this change?
But yeah according to PEP8 the only best practice is consistency. So I guess since we're using double quotes in most places, lets convert all these cases with single to double quotes like here?

File: stable_baselines3/parallel_envs.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import os
+import sys
+import diambra.arena
+from diambra.arena.stable_baselines3.make_sb3_env import make_sb3_env
+from stable_baselines3 import PPO
+
+if __name__ == '__main__':
```
Comment (discordianfish) [inline stable_baselines3/parallel_envs.py:7]: e.g here ' -> "

File: stable_baselines3/saving_loading_evaluating.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+from stable_baselines3 import A2C
+from stable_baselines3.common.evaluation import evaluate_policy
+
+if __name__ == "__main__":
+
+    # Create environment
+    env = diambra.arena.make("doapp", {"hardcore": True, "frame_shape": [128, 128, 1]})
+
+    # Instantiate the agent
+    agent = A2C('CnnPolicy', env, verbose=1)
```
Comment (discordianfish) [inline stable_baselines3/saving_loading_evaluating.py:11]: ..and here

File: stable_baselines3/training.py [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    learning_rate = linear_schedule(ppo_settings["learning_rate"][0], ppo_settings["learning_rate"][1])
+    clip_range = linear_schedule(ppo_settings["clip_range"][0], ppo_settings["clip_range"][1])
+    clip_range_vf = clip_range
+    batch_size = ppo_settings["batch_size"]
+    n_epochs = ppo_settings["n_epochs"]
+    n_steps = ppo_settings["n_steps"]
+
+    if model_checkpoint == "0M":
+        # Initialize the agent
+        agent = PPO('MultiInputPolicy', env, verbose=1,
```
Comment (discordianfish) [inline stable_baselines3/training.py:65]: ' -> "

File: stable_baselines/agent.py [hunk 5]
```diff
@@ -4,13 +4,10 @@
...
 import argparse
-base_path = os.path.dirname(os.path.abspath(__file__))
-sys.path.append(os.path.join(base_path, '../'))
-from make_sb_env import make_sb_env
-
+from diambra.arena.stable_baselines.make_sb_env import make_sb_env
 from stable_baselines import PPO2
 
-# diambra run -g python stable_baselines/v2/scripts/evaluation.py --cfgFile $PWD/stable_baselines/v2/scripts/cfg_files/doapp/sr6_128x4_das_nc.yaml  --trainedModel "25M"
+# diambra run -g python stable_baselines/agent.py --cfgFile $PWD/stable_baselines/cfg_files/doapp/sr6_128x4_das_nc.yaml  --trainedModel "25M"
```
Comment (discordianfish) [inline stable_baselines/agent.py:10]: What about moving this with a few more words to a docstring at the top of the file?
```
"""This is a example agent based on stable baselines.

Usage:
diambra run -g python stable_baselines/agent.py --cfgFile $PWD/stable_baselines/cfg_files/doapp/sr6_128x4_das_nc.yaml --trainedModel "25M"
"""

Comment (discordianfish) [reply stable_baselines/agent.py:10]: (I think it's just better for consistency but we can also use that eventually to generate docs from it automatically)

File: stable_baselines3/dict_obs_space.py (hunk 1 above)
Comment (alexpalms) [reply stable_baselines3/dict_obs_space.py:5]: yeah actually I think this happened with the auto corrector of visual code

File: stable_baselines3/parallel_envs.py (hunk 2 above)
Comment (alexpalms) [reply stable_baselines3/parallel_envs.py:5]: yes

File: stable_baselines3/saving_loading_evaluating.py (hunk 3 above)
Comment (alexpalms) [reply stable_baselines3/saving_loading_evaluating.py:11]: thanks yes

File: stable_baselines3/training.py (hunk 4 above)
Comment (alexpalms) [reply stable_baselines3/training.py:64]: noted
